pkg/parser/linter.go

```diff
@@ -95,16 +97,19 @@ func (l *PackageLinter) Lint(pkg *Package) error {
 
 // Or checks that at least one of the passed linter functions does not return an
 // error.
-func Or(a, b ObjectLinterFn) ObjectLinterFn {
+func Or(linters ...ObjectLinterFn) ObjectLinterFn {
 	return func(o runtime.Object) error {
-		if a == nil || b == nil {
-			return errors.New(errNilLinterFn)
-		}
-		aErr := a(o)
-		bErr := b(o)
-		if aErr == nil || bErr == nil {
-			return nil
+		var errSet string
+		for _, l := range linters {
+			if l == nil {
+				return errors.New(errNilLinterFn)
+			}
+			err := l(o)
+			if err == nil {
+				return nil
+			}
+			errSet += err.Error() + ", "
 		}
-		return errors.Errorf(errOrFmt, aErr, bErr)
+		return errors.Errorf(errOrFmt, strings.TrimSuffix(errSet, ", "))
```

Comment:
Nit: Could you just build a slice of strings then join them, to avoid needing to trim?

Reply:
Ah I knew there was a better way to do this, thanks!